CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+### v2.0.0-beta3 -> v2.0.0-beta4
+
+#### Features
```

Comment:
Can you please remove the empty sections from this document? Only "Documentation" actually has content.